playground-remote-inference.md

```diff
@@ -0,0 +1,72 @@
+# Remote inference in Playground
```

Comment:
For all screenshots, avoid personal/org information...
As well as non-aitk extensions (icon, sidebar)...

Reply:
Removed personal / org info and other extensions in sidebar